src/main/java/com/diffblue/deeptestutils/Reflector.java

```diff
@@ -425,21 +425,18 @@ private static void makePublic(final CtClass c) {
         }
 
         // In the case of an abstract class, search any constructor in the
-        // superclass and add an empty one with the same types
+        // superclass and add an empty one with the same types. The class must
+        // have a constructor.
         if (!cl.isInterface()) {
```

Comment:
Nit: the logic might be slightly easier to follow with the if and else cases swapped, i.e. "if it's an interface, do a simple thing, otherwise do something a bit more complicated" is a bit clearer than its current state of "if it's not an interface, do some complicated things, otherwise (i.e. if it _is_ an interface) just make a default constructor".